Shared/Mobile-Rounds.ViewModels/Regular/ReadingInput/ReadingInputScreenViewModel.cs

```diff
@@ -20,20 +22,20 @@ public class ReadingInputScreenViewModel : BaseViewModel
         /// Initializes a new instance of the <see cref="ReadingInputScreenViewModel"/> class.
         /// Represents the full view model for the page.
         /// </summary>
-        public ReadingInputScreenViewModel(string regionName, string stationName, string reads, Guid stationId)
+        public ReadingInputScreenViewModel(RegionModelSource region, StationModel station, string reads)
         {
             var regionNav = new AsyncCommand((obj) =>
             {
                 Navigator.Navigate(Shared.Navigation.NavigationType.StationSelect);
             });
-            this.Crumbs.Add(new BreadcrumbItemModel(regionName, regionNav));
-            this.Crumbs.Add(new BreadcrumbItemModel(stationName));
+            this.Crumbs.Add(new BreadcrumbItemModel(region.Name, region.Navigate));
+            this.Crumbs.Add(new BreadcrumbItemModel(station.Name, station.Navigate));
```

Comment:
@ChrisWillette Here is where I am using the navigation stuff from the breadcrumb. The `region.Navigate` property is what will get called when the region name is specified.